### PR TITLE
Bugfix/cpas mutation

### PR DESCRIPTION
**Pull request by dendoesit**
## What?

Start checkout mutation is being called twice inside connect payment when the button is called resulting in a second StartCheckout mutation that ends up in failure showing the error modal for a couple of seconds and causing a Member mutation failure.

## Why?

I believe the start checkout mutation takes too long so the even is fired twice. 

This may not be the best approach so happy to hear other options to fix this. 

Now : 

image

### CODE REVIEW

File: src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx [hunk 1]
```diff
@@ -242,6 +246,8 @@ export const CheckoutPayment = ({
         })
         throw new Error('Manual Review required')
       }
+      setIsCheckoutLoading(false)
```
Comment (robinandeer) [inline src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx:249]: should this be in a "finally"-block?

File: src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx [hunk 2]
```diff
@@ -224,6 +227,7 @@ export const CheckoutPayment = ({
           eventName: EventName.CheckoutErrorBasicError,
         })
         setIsError(true)
+        setIsCheckoutLoading(true)
```
Comment (robinandeer) [inline src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx:230]: why "true" - shouldn't it be false?

**Review by robinandeer — COMMENTED** [review]
Nice find - let's get it out

File: src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx (hunk 2 above)
Comment (dendoesit) [reply src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx:230]: yeah, meant to write false.

File: src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx (hunk 1 above)
Comment (dendoesit) [reply src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx]: good point. will add.

**Review by robinandeer — APPROVED** [review]
Nice, I hope this will do the trick :)

**Pull request merged**